src/System.Runtime/tests/System/Reflection/TypeInfoTests.cs

```diff
@@ -0,0 +1,223 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.IO;
+using System.Linq;
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public class TestType : Stream
```

Comment:
I wanted something with a large number of methods. I will change this to a custom type.